README.md

```diff
@@ -172,6 +172,15 @@ Uptime card is highly customizable.
 | animation | `none`/`raise`/`slide`/`reveal` | `raise` | v0.9.0 | The initialization animation **[More info](https://github.com/dylandoamaral/uptime-card#initialization)**. |
 | duration  |             number              |   0.5   | v0.9.0 | The initialization animation duration.                                                                     |
 
+### Clipping config
+
+**Parent key:** `init`
```

Comment:
Is it possible the parent key should be `clip`?

Reply:
It totally should thank you !